modules/deploy/pages/setting-up-dr-cluster.adoc

```diff
@@ -75,38 +110,37 @@ image::ROOT:{image-sgw-xdcr-dr-same-regn-in-recovery}[,{std-image-size}]
 
 // end::clusters-in-same-region[]
 
-
 === Clusters in Different Regions or Data Centers
 // tag::clusters-in-diff-region[]
 
 This model caters for situations where the Active and Disaster Recovery clusters are in different regions or data centers.
 Although the model has a separate Sync Gateway cluster attached to the Disaster Recovery cluster, it maintains the deployment of Sync Gateway at only one end of the XDCR replication.
-The optional optimization step will ensure that there is no downtime during the activation stage.
+The optional optimization step verifies that there is no downtime during the activation stage.
 
 
 .Set Up
 To set up and maintain a disaster recovery cluster - see: <<fig-dr-diff-regn-setup>>:
 
-. [_Optional step -- for optimization_] Turn on _Sync Gateway_ in the Disaster Recovery cluster just long enough to create indexes.
+. [Optional step -- for optimization] Turn on Sync Gateway in the Disaster Recovery cluster just long enough to create indexes.
```

Comment:
See above, I'd switch the recommendation here turn on Sync Gateway with offline: true which will asynchronously create indexes.